docs/mkdocs/en/memory/mysql.md

```
@@ -0,0 +1,57 @@
+- `WithMySQLInstance(name)`: Use pre-registered MySQL instance
+- `WithSoftDelete(enabled)`: Enable soft delete (default false)
+- `WithTableName(name)`: Custom table name (default "memories")
+- `WithMemoryLimit(limit)`: Memory limit per user
+- `WithCustomTool(toolName, creator)`: Register custom tool
+- `WithToolEnabled(toolName, enabled)`: Enable/disable tool
+- `WithExtraOptions(...options)`: Extra options passed to MySQL client
+- `WithSkipDBInit(skip)`: Skip table initialization (for users without DDL permissions)
+
+**DSN example**:
+
+```
```

markdownlint-cli2 (0.23.2)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)